migrations/capcons/capabilitymigration.go

```diff
@@ -39,7 +39,8 @@ type CapabilityMigrationReporter interface {
 	)
 	MissingBorrowType(
 		accountAddress common.Address,
-		addressPath interpreter.AddressPath,
+		targetPath interpreter.AddressPath,
+		storedPath Path,
```

Comment:
Maybe combine `accountAddress` and `storedPath` into a `interpreter.AddressPath`. Same below

Reply:
Updated in 0f12301ee4a7ee4cb8db24aca684d66a6e17440b